app/views/uploads/new.html.erb

```diff
@@ -9,17 +9,52 @@
     <div class="col-xs-8">
       <%= render 'form' %>
     </div>
-    <% if @requirements&.any? %>
-      <div class="col-xs-4">
-        <div class="alert alert-info" role="alert">
-          <h4 class="alert-heading">CSV Data Requirements</h4><br />
-          <ul>
-            <% @requirements.each do |validations| %>
-              <li><%= validations %></li>
+        <% if @requirements&.any? %>
```

Comment:
Should this be broken out into a partial since it is displayed in both new and show?